test/e2e/conftest.py

```diff
@@ -21,6 +22,17 @@ def fixture_test_client() -> TestClient:
     return TestClient(app, headers={"Authorization": f"Bearer {VALID_ACCESS_TOKEN}"})
 
 
+@pytest.fixture(name="add_usage_statuses_to_database", autouse=True, scope="session")
+def fixture_add_usage_statuses_to_database():
+    """
+    Fixture to add usage statuses to the usage_statuses collection in the database before any tests run
+    """
+    usage_statuses_collection = get_database().usage_statuses
+    usage_statuses_collection.delete_many({})
+    usage_statuses_collection.insert_many(USAGE_STATUSES)
+    yield
```

Comment:
Could this be moved to the `test_usage_status.py` `e2e` test module? Unless I am wrong that's the only module that requires it?

Reply:
Its also needed to do the item tests given that the check is in place for the create.

Reply:
Ah yes, sorry, I didn't realise that.